projects/canviewer/config.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Shutdown Nodes
+ss_bspd:
+ss_is:
+ss_bots:
+ss_estop_driver:
+ss_hvd:
+ss_hvd_conn:
+ss_mpc:
+ss_bms:
+ss_imd:
+ss_tsms:
+---
+# Vehicle Values
+air_p_status:
+air_n_status:
+ready_to_drive:
+pack_voltage:
+max_temperature:
+min_temperature:
+D1_DC_Bus_Voltage:
+---
+# Vehicle States (state and fault)
+air_control_critical:
```

Comment:
Why are these organized by message name, where the previous signals aren't?

Reply:
The message is used both by the GUI (so it can display the message name on the left column of the states table) and by the CAN receiver (so it knows which message a signal comes from).

Theoretically we could eliminate the need for it if:
- We made a dictionary for all of the signals and their corresponding messages from the DBC, eliminating the second functionality
- We added some restructuring between the dictionary that we get after filling in CAN values and the one we send to the GUI so it knows where to display all the values (so for example it knows to place `air_state` in the AIR control row, or for that matter, even knows the AIR control row exists)

In my opinion, that's more work than it would be worth, and syntactically it does make some sense that you specify that you want the state/fault from a certain component as opposed to the others where the source of the message isn't as relevant (eg  it would be irrelevant to the user that ss_bots comes from the BSPD board, but saying you want the state signal from the throttle board is justifiable)

Reply:
Ah I think I see. I guess what I mean is that the system should be unified. In other words, you should either have to specify the message name _and_ the signal name, or just the signal name, not a mixture of both. Ya dig?

Also, I think best practice is to not have duplicate signal names, so every signal name should be unique. If it isn't (like maybe I mixed two things up), we should rename them, like from `state` to `throttle_state` or something like that. You can do this in this PR if you want or I can do it in a separate one.

Reply:
If we keep standardization of state and fault names, how's this for a solution - we can just specify the message name, and use the DBC to pull the names of the signals. Something like this:
```
#Vehicle States/Faults
air_control_critical:
bms:
throttle:
```
This (to me at least) makes logical sense because the user can specify they want the state and fault from say the throttle board, which is more pertinent than what the actual signal name is. Not technically consistent, but it is much simpler.

We could also do this in reverse, where we take the signal names, and use the DBC to find what the names of the messages are.

Essentially, the dictionary that gets passed into the GUI would be the same, we would just need to do some leg work with the DBC at the start of the program to fill in the gaps and build what is specified in the YAML now.

As far as uniqueness of signal names - they are technically unique right now, although I agree that just calling something `state` is terrible practice - pretty sure that was me when I wrote the throttle YAML. I'll change that in the next rev.